css/Components/Title.scss

```diff
@@ -5,18 +5,26 @@ $Title-font-size: 72px;
 $Title-line-height: 91px;
 $Title-font-size-mobile: 39px;
 $Title-line-height-mobile: 52px;
-$Title-font-family: "MatrixIIOT";
-$Title-font-weight: 800;
+$Title-font-family: "matrix-ii";
+$Title-font-weight: bold;
```

Comment:
I'm not sure how this works in gatsby, but since each Scss file is imported in it's own react component, do we really need to namespace all the variables here, or would just `$font-weight` work?

If it does, I would say that the namespacing only hinders readability